deadmatterupdates.json

```diff
@@ -0,0 +1,57 @@
+[
+    {
+        "UpdateStageName":"SteamCMD Download",
+        "UpdateSourcePlatform":"All",
+        "UpdateSource":"SteamCMD",
+        "UpdateSourceData":"2584780",
+        "UpdateSourceArgs":"2184150",
+        "ForceDownloadPlatform":"Windows"
+    },
+    {
+        "UpdateStageName":"SteamCMD Component Download",
+        "UpdateSourcePlatform":"All",
+        "UpdateSource":"SteamCMD",
+        "UpdateSourceData":"1007",
+        "UpdateSourceTarget":"{{$FullBaseDir}}DeadMatter/Binaries/Win64",
+        "ForceDownloadPlatform":"Windows"
+    },
+    {
+        "UpdateStageName":"Config Directory Creation",
+        "UpdateSourcePlatform":"All",
+        "UpdateSource":"CreateDirectory",
+        "UpdateSourceArgs":"{{$FullBaseDir}}DeadMatter/Saved/Config/WindowsServer",
+        "OverwriteExistingFiles":false
+    },
+    {
+        "UpdateStageName":"Game.ini Config Download",
+        "UpdateSourcePlatform":"All",
+        "UpdateSource":"FetchURL",
+        "UpdateSourceData":"https://raw.githubusercontent.com/winglessraven/AMPTemplates/deadmatter-wine/deadmattergame.ini",
+        "UpdateSourceArgs":"Game.ini",
+        "UpdateSourceTarget":"{{$FullBaseDir}}DeadMatter/Saved/Config/WindowsServer/",
+        "OverwriteExistingFiles":false
+    },
+    {
+        "UpdateStageName":"Initialise Wine",
+        "UpdateSourcePlatform":"Linux",
+        "UpdateSource":"Executable",
+        "UpdateSourceData":"/bin/bash",
+        "UpdateSourceArgs":"-c 'WINEPREFIX=\"{{$FullRootDir}}.wine\" WINEARCH=win64 /usr/bin/wineboot --init --update'"
+    },
+    {
+        "UpdateStageName":"Winescript Download",
+        "UpdateSourcePlatform":"Linux",
+        "UpdateSource":"FetchURL",
+        "UpdateSourceData":"https://raw.githubusercontent.com/winglessraven/AMPTemplates/deadmatter-wine/deadmatterwinescript.sh",
```

Comment:
This will need to be repointed to the CC repo before merge.

Reply:
Not needed - removed in latest commit